mdata/notifierNsq/notifierNsq.go

```diff
@@ -52,13 +53,11 @@ func New(instance string, metrics mdata.Metrics, idx idx.MetricIndex) *NotifierN
 		log.Fatal(4, "nsq-cluster failed to create NSQ consumer. %s", err)
 	}
 	c := &NotifierNSQ{
-		in:       make(chan mdata.SavedChunk),
-		instance: instance,
-		Notifier: mdata.Notifier{
-			Instance: instance,
-			Metrics:  metrics,
-		},
-		idx: idx,
+		instance:      instance,
+		in:            make(chan mdata.SavedChunk),
+		metrics:       metrics,
+		idx:           idx,
+		createMissing: true,
```

Comment:
if both notifierKafka and notifierNSQ are hard coding createMissing=true, then we should just remove it.